plugins/n-test/README.md

```diff
@@ -0,0 +1,34 @@
+# @dotcom-tool-kit/n-test
+
+A plugin to run smoke tests as part of your CircleCI workflow using the [n-test](https://github.com/Financial-Times/n-test) package.
+
+## Installation
+
+With Tool Kit [already set up](https://github.com/financial-times/dotcom-tool-kit#installing-and-using-tool-kit), install this plugin as a dev dependency:
+
+```sh
+npm install --save-dev @dotcom-tool-kit/n-test
+```
+
+And add it to your repo's `.toolkitrc.yml`:
+
+```yaml
+plugins:
+    - '@dotcom-tool-kit/n-test'
+```
+
+## Options
```

Comment:
**thought**: these probably could be a bit more descriptive but unfortunately the original documentation is pretty limited too. maybe we should link to the [file](https://github.com/Financial-Times/n-test/blob/main/tasks/smoke.js) where we've pulled these descriptions from in case they're updated at some point?